sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseCriteriaAndSorting.java

```diff
@@ -0,0 +1,27 @@
+package de.symeda.sormas.api.caze;
+
+import de.symeda.sormas.api.utils.SortProperty;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class CaseCriteriaAndSorting implements Serializable {
```

Comment:
Since we'll need this for basically every list at some point (contacts, samples, tasks, and so on), is it possible to do it in a more generic way? Something like:

`public class CriteriaWithSorting<T extends BaseCriteria> implements Serializable`?